src/microbe_stage/organelle_components/CiliaComponent.cs

```diff
@@ -81,6 +94,11 @@ public override void UpdateSync()
             // better to do things correctly here as this is newer code...
             SetSpeedFactor(targetSpeed);
         }
+
+        if (attractorAreaShape != null)
+        {
+            attractorAreaShape.Disabled = organelle!.ParentMicrobe!.State != Microbe.MicrobeState.Engulf;
```

Comment:
Should the pull effect be disabled if this microbe this cilia is on gets engulfed? Otherwise wouldn't the engulfing cell move around with the extra gravity pull area causing quite many potential issues?

Reply:
I think cells have their engulfment mode automatically disabled (thus disables pulling) when they're engulfed so this conditional kind of kills two birds with one stone already, but an extra check there would still be nice.

Reply:
Oh yeah, that's probably true. An explicit check might be better anyway, as we just found out I did not immediately think about that.

Reply:
Done.